src/utils.jl

```diff
@@ -0,0 +1,43 @@
+"""
+    ignore() do
+        ...
+    end
+
+Tell ChainRulesCore to ignore a block of code. Everything inside the `do`
+block will run on the forward pass as normal, but ChainRulesCore won't try to
+differentiate it at all. This can be useful for e.g. code that does logging of
+the forward pass.
+
+Obviously, you run the risk of incorrect gradients if you use this incorrectly
+"""
+ignore(f) = f()
```

Comment:
Do we want to call this something else, so that we don't have a clash with Zygote?
Or does Zygote not export this anyway?

We might want to export this, but as something less clashing.
Like `ad_ignore`

Reply:
Zygote doesn't export it. I think it works quite well requiring it as a fully-qualified name, and `ChainRulesCore.ignore` is quite self-explanatory. I don't think there's a specific need to export it, if there's docs that show how to use it with the module prefix. If you do want to export it, `ad_ignore` or even `autodiff_ignore`?